provisioner.go

```diff
@@ -103,19 +104,23 @@ func (p *defaultProvisioner) Provision(ctx context.Context, d Dispatcher, appNam
 	}
 	s.Astilectron = &ProvisionStatusPackage{Version: VersionAstilectron}
 
+	if s.Electron == nil {
```

Comment:
I'd rather get an already initialized `ProvisionStatus` coming out of `p.ProvisionStatus(paths)`.

Could you move this logic to `func (p *defaultProvisioner) ProvisionStatus(paths Paths) (s ProvisionStatus, err error)` by initializing `s` at the very beginning?